internal/users/tests/tests.go

```diff
@@ -16,26 +19,33 @@ import (
 var (
 	//go:linkname defaultOptions github.com/ubuntu/authd/internal/users.defaultOptions
 	defaultOptions struct {
-		groupPath  string
-		gpasswdCmd []string
+		groupPath   string
+		gpasswdCmd  []string
+		getUsersCmd []string
 	}
 )
 
 // OverrideDefaultOptions allow to change groupPath and gpasswdCmd without using options.
 // This is used for tests when we don’t have access to the users object directly, like integration tests.
 // Tests using this can't be run in parallel.
-func OverrideDefaultOptions(t *testing.T, groupPath string, gpasswdCmd []string) {
+func OverrideDefaultOptions(t *testing.T, groupPath string, gpasswdCmd []string, getUsersCmd []string) {
 	t.Helper()
 
 	origin := defaultOptions
-	t.Cleanup(func() { defaultOptions = origin })
+	t.Cleanup(func() {
+		defaultOptions = origin
+	})
```

Comment:
I don’t think this formatting change for a one line brings anything, or was it an issue with a newer gci, as it passed in the past?

Reply:
I probably changed the formatting to add another option there, but don't remember it now :sweat_smile: I'll switch back to the one line form (I also prefer it as well)